packages/vps/spec

```diff
@@ -0,0 +1,8 @@
+---
+name: vps
+
+dependencies:
+  - golang_1.7
+
+files:
+- "**/*"
```

Comment:
Let's make sure to only copy files we need instead of `**/*` prefix `golang_1.7`